perception/traffic_light_classifier/package.xml

```diff
@@ -9,17 +9,20 @@
   <license>Apache License 2.0</license>
 
   <buildtool_depend>ament_cmake_auto</buildtool_depend>
-  <buildtool_depend>autoware_cmake</buildtool_depend>
   <buildtool_depend>wget</buildtool_depend>
 
+  <build_depend>autoware_cmake</build_depend>
```

Comment:
Why did you change the order of package?
It looks the same for me.

- before PR
```bash
  <buildtool_depend>ament_cmake_auto</buildtool_depend>
  <buildtool_depend>autoware_cmake</buildtool_depend>
  <buildtool_depend>wget</buildtool_depend>
```

- after PR
```bash
  <buildtool_depend>ament_cmake_auto</buildtool_depend>
  <buildtool_depend>wget</buildtool_depend>

  <build_depend>autoware_cmake</build_depend>
```